snapcraft/linters/library_linter.py

```diff
@@ -101,6 +104,23 @@ def run(self) -> List[LinterIssue]:
 
         return issues
 
+    def _find_deb_package(self, library_name: str) -> Optional[str]:
+        for path in glob.glob("/usr/lib/**", recursive=True):
+            if os.path.basename(path) == library_name:
+                try:
+                    output = subprocess.run(
```

Comment:
you are not mocking this anywhere, which explains why the tests fail, try in a clean environment, perhaps 20.04 and you will be able to reproduce

Reply:
The Spread Tests with google:ubuntu-22.04-64 is passed sometimes, so I think that it isn't a problem of my patch.

Reply:
Also, the TOX/Linters error is in a different file. I think that with a rebase it will pass. May I...?